agent/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghcr.io/openwallet-foundation/acapy-agent:py3.12-1.2.0
+
+USER root
+
+# install plugins as binaries
+RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@main#subdirectory=webvh
+RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@main#subdirectory=connection_update
+RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@main#subdirectory=basicmessage_storage
+RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@main#subdirectory=multitenant_provider
```

Comment:
Please target the tag appropriate to the base image version to avoid issues untested changes - see [this example](https://github.com/bcgov/traction/blob/b3fcf1145c5debae1593032b2c04c1845751a03f/plugins/docker/Dockerfile#L33-L36)

Reply:
Will do thanks